unstructured_ingest/v2/processes/connectors/kafka/kafka.py

```diff
@@ -157,7 +162,9 @@ async def run_async(self, file_data: FileData, **kwargs: Any) -> DownloadRespons
     def precheck(self):
         try:
             with self.get_consumer() as consumer:
-                cluster_meta = consumer.list_topics(timeout=self.index_config.timeout)
+                # timeout needs at least 3 secs, more info:
```

Comment:
We should enforce this at the field level by setting the greater-than parameters: 
```python
timeout: Optional[float] = Field(default=3.0, description="polling timeout", ge=3)
```